cryptus-app/components/profile/creator_profile/creator_profile_infos/edit_profile_button/edit_profile_button.module.scss

```diff
@@ -0,0 +1,24 @@
+@use "/styles/variables" as v;
+
+.container {
+  flex: 1 1 auto;
+
+  display: flex;
+  align-items: end;
+}
+
+.editbutton {
```

Comment:
spacing, rounding and color is not what was in the document:
![image](https://user-images.githubusercontent.com/32850285/152713417-907a8807-e012-4430-91c2-d70a3e7d1abf.png)